pkg/api/api.go

```diff
@@ -260,6 +266,12 @@ func Router() *gin.Engine {
 				obMap.PATCH("topic/:topic/license", PatchObligationMap)
 				obMap.PUT("topic/:topic/license", UpdateLicenseInObligationMap)
 			}
+			oidcClient := authorizedv1.Group("/oidcClients")
+			{
+				oidcClient.GET("", middleware.RoleBasedAccessMiddleware([]string{"ADMIN"}), GetUserOidcClients)
+				oidcClient.POST("", middleware.RoleBasedAccessMiddleware([]string{"ADMIN"}), AddOidcClient)
+				oidcClient.DELETE("", middleware.RoleBasedAccessMiddleware([]string{"ADMIN"}), RevokeClient)
```

Comment:
These endpoints should be allowed to called by all users so they can add their client-ids.